### DISCUSSION TITLE
can't work  v1.7.4924

### DISCUSSION THREAD

**Discussion by carl-lou** [community/community#11292] [Copilot Conversations] [answered]
my copilot is not work

my version is  v1.7.4924

![image](https://user-images.githubusercontent.com/46988469/153378849-23df5dfb-5cb3-4882-af58-81ee3fdb58c8.png)

My colleague executing the same command can be used

**Comment by davecheney** [comment]
I'm sorry you are having problems. Can you please describe what you tried to do, what happened when you tried, and what you expected to happen. Thank you.

**Reply by carl-lou** [reply]
I forgot to log in the account, no authorization, hahaha
It's ready to use now

我忘记登录账号了，没授权，哈哈哈
现在可以用了

**Comment by carl-lou** [comment, marked as answer]
I forgot to log in the account, no authorization, hahaha


我忘记登录账号了，没授权，哈哈哈

**Reply by davecheney** [reply]
Cool. I'm glad you figured it out.
